### PR TITLE
throw DecodeException in Base64Codec

### PR DESCRIPTION
**Pull request by rubanm**

### CODE REVIEW

File: core/src/main/java/com/twitter/elephantbird/mapreduce/input/Base64Codec.java [hunk 1]
```diff
@@ -170,8 +172,16 @@
    * @return The decoded array of bytes. May be of length 0. Will be <code>null</code> if the legal characters
    * (including '=') isn't divideable by 4. (I.e. definitely corrupted).
    */
-  public final static byte[] decode(byte[] sArr, int sOff, int sLen)
+  public final static byte[] decode(byte[] sArr, int sOff, int sLen) throws DecodeException
   {
+    try {
+      return doDecode(sArr, sOff, sLen);
+    } catch (Exception e) {
```
Comment (isnotinvain) [inline core/src/main/java/com/twitter/elephantbird/mapreduce/input/Base64Codec.java]: is RuntimeException enough?

Comment (rubanm) [inline core/src/main/java/com/twitter/elephantbird/mapreduce/input/Base64Codec.java]: Yes, changed to RuntimeException.

**Comment by ianoc** [comment]
LGTM

**Comment by isnotinvain** [comment]
+1

**Pull request merged**